src/main/java/org/pac4j/jax/rs/pac4j/JaxRsApplicationLogoutLogic.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.jax.rs.pac4j;
+
+import org.pac4j.core.engine.DefaultApplicationLogoutLogic;
+
+/**
+ * @author Victor Noel - Linagora
+ * @since 1.1.1
+ * 
+ */
+public class JaxRsApplicationLogoutLogic<C extends JaxRsContext> extends DefaultApplicationLogoutLogic<Object, C> {
```

Comment:
I wish we could get rid of such simple customized classes. And we have three of them for which we just set the same specific `JaxRsProfileManager`.

I have a proposal: Add a `profileManager` at the `Config` level, `null` by default. Logics would use it if it's not `null` or the generic `ProfileManager` otherwise.
In JAX-RS and other implementations that require a specific profile manager, a check could be made in the filters or somewhere else to set the specific one (`JaxRsProfileManager`) if none is already defined (in the `Config`).

Does that make sense? @millross for Vertx also?